pandas/tseries/tests/test_timeseries.py

```diff
@@ -3115,6 +3151,7 @@ def test_datetime64_with_DateOffset(self):
             # assert these are equal on a piecewise basis
             offsets = ['YearBegin', ('YearBegin', {'month': 5}), 'YearEnd',
                        ('YearEnd', {'month': 5}), 'MonthBegin', 'MonthEnd',
+                       'SemiMonthEnd', 'SemiMonthBegin',
```

Comment:
in the list of times to test (`s = klass([Timestamp() ...])` a few lines up), add a date that lands on the semi-month offset, e.g. `2000-01-15`